inc/core/MicroBitConfig.h

```diff
@@ -130,7 +136,7 @@ extern uint32_t __etext;
 // For standard S110 builds, this should be word aligned and in the range 0x300 - 0x700.
 // Any unused memory will be automatically reclaimed as HEAP memory if both MICROBIT_HEAP_REUSE_SD and MICROBIT_HEAP_ALLOCATOR are enabled.
 #ifndef MICROBIT_SD_GATT_TABLE_SIZE
-#define MICROBIT_SD_GATT_TABLE_SIZE             0x300
+#define MICROBIT_SD_GATT_TABLE_SIZE             0x400
```

Comment:
@microbit-sam I think this service should be included by default, but can we check if this additional 256 bytes of RAM allocation is really needed? RAM is very tight with BLE enabled, and I'd like to not take away more for applications than strictly necessary.

Reply:
Note also the RAM allocated in member variables that are only used in pairing mode.